docs/config.md

```diff
@@ -267,3 +268,17 @@ Disables the p2p-circuit relay transport.
 - `EnableRelayHop`
 Enables HOP relay for the node. If this is enabled, the node will act as
 an intermediate (Hop Relay) node in relay circuits for connected peers.
+
+## Profiles
```

Comment:
Hrm... i'm not sure this belongs here. Maybe put it at the top of the document? to distinguish it from just looking like another config field

Reply:
Moved it to the header, makes a bit more sense now